api/kessel/inventory/v1beta1/relationships/k8spolicy_ispropagatedto_k8scluster_detail.proto

```diff
@@ -16,15 +18,11 @@ message K8SPolicyIsPropagatedToK8SClusterDetail {
     COMPLIANT = 3;
   }
 
-  // The resource ID assigned to the resource by Kessel Asset Inventory. A
-  // reporter alias may also be used as a resource ID using the format:
-  // \"hcrn:<reporter_data.reporter_type>:<reporter_data.reporter_id>::<reporter_data.local_resource_id>\"
-  int64 k8s_policy_id = 225679544;
+  // The resource ID assigned to the resource by Kessel Asset Inventory. 
+  int64 k8s_policy_id = 225679544 [ (google.api.field_behavior) = OUTPUT_ONLY ];
 
-  // The resource ID assigned to the resource by Kessel Asset Inventory. A
-  // reporter alias may also be used as a resource ID using the format:
-  // \"hcrn:<reporter_data.reporter_type>:<reporter_data.reporter_id>::<reporter_data.local_resource_id>\"
-  int64 k8s_cluster_id = 240280960;
+  // The resource ID assigned to the resource by Kessel Asset Inventory. 
+  int64 k8s_cluster_id = 240280960 [ (google.api.field_behavior) = OUTPUT_ONLY ];
 
   Status status = 355610639;
```

Comment:
Unrelated to this PR:

If this is part of a request data and we want to ensure we only need valid `Status` values (minus the unspecified) we need to update this to:
```suggestion
  Status status = 355610639; [ (buf.validate.field).enum = {not_in: [0], defined_only: true} ]
```

This will require this import:
```
import "buf/validate/validate.proto";
```

Reply:
thx.  Will add